react/states_and_effects/more_on_state.md

```diff
@@ -51,11 +51,15 @@ function Person() {
 }
 ~~~
 
+
 <div class="lesson-note lesson-note--tip" markdown="1">
 
-#### Nested objects and arrays in state
+#### Objects and arrays in state
+
+In the above example, notice how we _create_ a new object, and then copy the existing state values into the new object while providing a new value for `age`.
+That is because, besides the unpredictability that comes up from changing the state manually, if we don't provide a new object to `setState` it is not guaranteed to re-render the page. Therefore we should always provide a new Object for `setState` to trigger a re-render. `setState` uses [Object.is](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is) to determine if the previous state is the same.
```

Comment:
> besides the unpredictability that comes up from changing the state manually

This sentence sounds very vague to me: what does changing state manually mean?

Reply:
By manually, I meant without relying on setState.

I could change it to something like
> besides the unpredictability that comes up from changing the state without the use of setState function

Or I could remove it all together, that will surely make the whole paragraph more concise.

Reply:
I vote remove it.